### DISCUSSION TITLE
Can ESPHome-econet be used without home assistant?

### DISCUSSION THREAD

**Discussion by ericmagaha** [esphome-econet/esphome-econet#426] [General]
Hello,

I just installed a Rheem hot pump water heater and I am looking for ways to get more data out of it.  I came across this project and it seems like an awesome way to do that.  However, I do not use Home Assistant.  If I set up ESPHome-econet on the recommended hardware is there a way to query for the data i.e. http, CLI etc?

I see that esphome can be run in host mode -> https://esphome.io/components/host.

Could I run this on a raspberry pi in host mode with a usb to 485 converter like this -> https://www.amazon.com/dp/B07B416CPK

and then run ESPHome dashboard -> https://esphome.io/guides/getting_started_command_line#bonus-esphome-dashboard

Thanks in advance!

**Comment by tronikos** [comment]
https://www.youtube.com/watch?v=0XH61uCqMDw
